pkg/porter/build.go

```diff
@@ -240,6 +240,17 @@ func (p *Porter) prepareDockerFilesystem() error {
 		}
 	}
 
+	// Create the local outputs directory, if it doesn't already exist
+	outputsDir, err := p.Config.GetOutputsDir()
+	if err != nil {
+		return errors.Wrap(err, "unable to get outputs directory")
+	}
+
+	err = p.FileSystem.MkdirAll(outputsDir, 0755)
```

Comment:
I'm a bit unsure of the code's intent right here? This function is meant to create files on the local file system in pwd/.cnab in preparation before building the invocation image.

If we need to make the outputs directory ahead of time, then the code should be doing this:

```
p.FileSystem.MkdirAll(filepath.Join(build.LOCAL_APP, "outputs"), 0755)
```

Otherwise if your intent was to make sure the outputs directory in PORTER_HOME exists before the bundle is run (which is what this code is doing), then it's in the wrong spot. It should be located in maybe `pkg/porter/options.go applyDefaultOptions` or really a new function that is executed in each function before calling the cnab provider functions for install/upgrade/etc.

Reply:
You know, I think we're okay with not ensuring the outputs directory exists before a bundle is run.  The logic around setting up the volume mount (if driver is the default Docker driver) is sufficient (will create `${PORTER_HOME}/outputs` if it doesn't already exist, in addition to the `<claim name>` subdir.)

So, I've removed for now.